scattr/workflow/Snakefile

```diff
@@ -69,12 +68,10 @@ include: "rules/qc.smk"
 
 rule all:
     input:
-        tck_files=expand(
-            rules.filtered_tck2connectome.output.sl_assignment,
-            zip,
-            **subj_zip_list
+        tck_files=inputs["T1w"].expand(
+            rules.filtered_tck2connectome.output.sl_assignment
```

Comment:
This going to expand over all the wildcards in `"T1w"`, not just subject and session like it did before. Is that intentional?

Reply:
Sort of - would it expand over those additional wildcards if they aren’t present in the file though?

Reply:
I _think_ `expand` errors out if you give it wildcards not in the path it's trying to expand, but am not sure

Reply:
It does not - at least not when I ran this workflow. Maybe this was unintentional and it should be throwing an error.

Reply:
Took a look at the snakemake source and it seems like you're right, I'm not sure why I thought that

Reply:
I think there is an open PR or issue on the snakebids side for handling partial expansions that might be where this thought came from?

In any case, do we still want to make a change here or leave as is?

Reply:
If you're satisfied that it works as is, feel free to just leave it

Reply:
I will check the outputs again before resolving this to make sure the naming and stuff are still right.